grype/presenter/sarif/presenter_test.go

```diff
@@ -0,0 +1,252 @@
+package sarif
+
+import (
+	"bytes"
+	"flag"
+	"regexp"
+	"testing"
+
+	"github.com/sergi/go-diff/diffmatchpatch"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/anchore/go-testutils"
+	"github.com/anchore/grype/grype/match"
+	"github.com/anchore/grype/grype/pkg"
+	"github.com/anchore/grype/grype/presenter/models"
+	"github.com/anchore/grype/grype/vulnerability"
+	"github.com/anchore/stereoscope/pkg/imagetest"
+	syftPkg "github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/source"
+)
+
+var update = flag.Bool("update", true, "update the *.golden files for json presenters")
```

Comment:
I'll update the golden files and flip this when the rest gets reviewed and it's ready to merge